PRIVACY.md

```diff
@@ -1,10 +1,8 @@
 # Privacy
 
 In general, the Safety Critical Rust Consortium adheres to the Rust Foundation's [privacy 
-policy](privacy-policy). 
```

Comment:
I actually just screwed up the brackets here. It should have been `[privacy policy][privacy-policy]`